docs/plugins/lifecycle.md

```diff
@@ -0,0 +1,66 @@
+# Lifecycle
+
+The SDK provides Application lifecycle events to control the running state. As an application you can listen for state changes and handle them accordingly.
+
+## Usage
+
+The SDK supports different lifecycle states to be used on the device. The lifecycle plugin makes the states available to the application through events.
+
+```js
+import { Lifecycle } from '@lightningjs/sdk'
+```
+
+## Available states
+
+The following states are available:
+
+| State         | Description                                 |
+| ------------- |:-------------------------------------------:|
+| init          | Application is initialising                 |
+| active        | Application is running, and in focus        |
+| background    | Application is running, not in focus        |
+| pause         | Application is paused, not in focus         |
+| close         | Application is closing and will be unloaded |
```

Comment:
We have been referring to this as `unload` internally. Does `close` have precedent elsewhere in the SDK?

Reply:
We do have the concept of unloaded, which means the JS + DOM + graphics is unloaded, however the application itself will never have that as a status. I.e. you can't know you're dead yourself. The MAF Store app manager will know once an app is unloaded.

The close method has been in our SDK for a while and is transitional state. It means either someone externally or the app itself wants to close, this means the app will need to do its cleanup routine and once the close is done the upper layers can "unload" it.

Reply:
makes sense to go with `close` to me, then.